pint/models/model_builder.py

```diff
@@ -57,6 +57,8 @@ def __init__(self, name, parfile = None):
         self.name = name
         self.model_instance = None
         self.param_inparF = None
+        self.param_unrcgnz = {}
```

Comment:
self.param_unrcgnz?   That is a terrible abbreviation...  Since this is internal code, I *highly* recommend a full and understandable name lik eself.param_unrecognized